bench/ring-rs/optimized/4prppzcttbsz5zvc.ll

```diff
@@ -506,8 +506,9 @@ define hidden { ptr, i64 } @_ZN4ring10arithmetic6bigint10elem_widen17h27713e316e
   store i64 %1, ptr %8, align 8
   %9 = getelementptr inbounds i8, ptr %2, i64 32
   %10 = load i64, ptr %9, align 8, !noundef !4
-  %.not = icmp ugt i64 %10, %3
-  br i1 %.not, label %11, label %.critedge
+  %.09 = tail call i8 @llvm.ucmp.i8.i64(i64 %3, i64 %10)
+  %switch = icmp ult i8 %.09, 2
```

Comment:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/1194#pullrequestreview-2238695484